manual_phasing_90_pulse.py

```diff
@@ -1,108 +0,0 @@
-from pyspecdata import *
-from proc_scripts import *
-from proc_scripts.load_data import postproc_dict
-from scipy.optimize import minimize,curve_fit,least_squares
-rcParams['lines.linewidth'] = 0.5
```

Comment:
again, this file should be added in a new/different pull request, not here

Reply:
I think the easiest way to do this will be 1) open a new branch off of master 2) `git checkout 16b309 manual_phasing_90_pulse.py` and 3) `git diff 16b309  FILEWITHPOSPROC` being very careful to copy over only the parts relevant to the manual phasing 90